js/index.js

```diff
@@ -82,11 +80,12 @@ function passwordScore(pass) {
     }
     return {score: score, error: error};
 }
-$('.signup-form').find('#password').keyup(function(ele) {
-	var scoreData = passwordScore($(this).val()),
+
+function passwordFieldKeyUp(t) {
```

Comment:
Also, https://github.com/fatimarafiqui/FirstAide-web/blob/e8fb4a1897b4c56c83a1f6922c3eb6b985938f34/js/index.js#L1 

The methods in this file assumes that the parameter will always be a non-null value. If a null value is passed in to these methods, and a function is called on the null value, the app will land in a Null Pointer Exception.  There should be a check to see if the element is not null before performing an action on it (such as calling a method on the element).